spec/models/mediaflux/time_spec.rb

```diff
@@ -23,7 +23,7 @@
         initial_tz = xml_snip.xpath("./@tz").text
 
         final_tz = instance.convert(xml_snip:)
-        expect(final_tz).to include("-04:00").or include("-05:00") #America/New_York changes based on daylights savings time
+        ["-04:00", "-05:00"].any? { |tz| final_tz.include?(tz) } #America/New_York changes based on daylights savings time
```

Comment:
Does this really fail the test if the timezone is not one of the expected values?  I would assume it just returns false, which would not fail the test

Reply:
Right, I can add an expectation check for the match to be true